Project/src/main/java/app/models/Editor.java

```diff
@@ -0,0 +1,52 @@
+package app.models;
```

Comment:
Whole class seems a little insignificant.

you have wrapper methods for methods that dont need a wrapper. Wrappers are usually used when extra functionality is required.